monitoring/monitorlib/scd_automated_testing/scd_injection_api.py

```diff
@@ -38,14 +38,15 @@ class FlightAuthorisationData(ImplicitDict):
 class InjectFlightRequest(ImplicitDict):
     ''' A class to hold the details of a test injection payload '''
     operational_intent: OperationalIntentTestInjection
-    flight_authorisation: FlightAuthorisationData
+    flight_authorisation: Optional[FlightAuthorisationData]
 
 
 class InjectFlightResult(str, Enum):
     Planned = 'Planned'
     Rejected = 'Rejected'
     ConflictWithFlight = 'ConflictWithFlight'
     Failed = 'Failed'
+    DryRun = 'DryRun'
```

Comment:
This is the API USSs under test will see, so I don't think we want an option to respond with "DryRun" from them

Reply:
I completely removed the dry run since it doesn't bring value at this point in time. It will be the topic of another PR if useful.